lms/djangoapps/homepage/custom_context_processors.py

```diff
@@ -1,31 +1,15 @@
 from edx_notifications.server.web.utils import get_notifications_widget_context
 
+from .constants import CONFIGS
+
 
 def notifications_configs(request):
     """
-        Context processor to set global configs for edx-notifications
+    Context processor to set global configs for edx-notifications
     :param request:
     :return: edx-notifications global configs
```

Comment:
I'll be passing over the last suggestion, since it is out of scope. To keep things consistent, I'll avoid this suggestions as well. Because I don't know the difference between 'context' and 'configs' and I don't think I need to know as a part of this ticket.

Reply:
done